content/sensu-go/6.6/observability-pipeline/observe-schedule/backend.md

```diff
@@ -1074,6 +1075,21 @@ sensu-backend start --etcd-client-cert-auth{{< /code >}}
 /etc/sensu/backend.yml example | {{< code shell >}}
 etcd-client-cert-auth: true{{< /code >}}
 
+<a id="etcd-log-level"></a>
+
+| etcd-client-log-level  |      |
+-------------|------
+description  | Logging level for the embedded etcd client: `panic`, `fatal`, `error`, `warn`, `info`, or `debug`. {{% notice note %}}
```

Comment:
Embedded is an overloaded term. One may assume that embedded etcd client only applies to the etcd client when embedded etcd is in use. This flag is used for the etcd client when used with embedded or external etcd.

Perhaps "internal" is more appropriate?

Reply:
Yes, I can change it to "internal"

A related question...is "embedded" appropriate for the etcd-log-level flag? We say "embedded" in that description table (see https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/#etcd-log-level) but I can change that to "internal" too if "embedded" is incorrect.

Reply:
@hillaryfraley yes, embedded is appropriate for the non-client etcd log level. We can only control the etcd server log level if it's running embedded within sensu-backend. External etcd servers are a separate process thus we have no control.

Reply:
Roger that, thank you @amdprophet !